iothub/device/src/MethodRequest.cs

```diff
@@ -6,20 +6,46 @@
 
 namespace Microsoft.Azure.Devices.Client
 {
+    /// <summary>
+    /// The data structure that represents a method request.
+    /// </summary>
     public sealed class MethodRequest
     {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="MethodRequest"/> class with without any method data and extra time for
```

Comment:
This is what I found by looking in code. Someone with more context please validate.

Reply:
the default value for timeouts here should be null -> not specified -> 30secs for response timeout and 0s for connect: https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-direct-methods#method-invocation